packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -96,7 +121,27 @@ function GridItem< Item >( {
 				justify="space-between"
 				className="dataviews-view-grid__title-actions"
 			>
-				<HStack className="dataviews-view-grid__primary-field">
+				<HStack
```

Comment:
What do you think of following the same approach as the table layout? This is: adding an extra div for the content, so the click target is the title text instead of the "cell area":

https://github.com/user-attachments/assets/48d8b8f6-72b8-42d9-a9ed-8370360a4686

Reply:
Fixed with 7a7964457a25010712721a454ee5d6b0c883ed63:

https://github.com/user-attachments/assets/670edb60-d237-47cf-a42c-065ee2aaf8a8